lib/rubocop/target_ruby.rb

```diff
@@ -102,7 +102,11 @@ def gemspec_filepath
           @config.traverse_directories_upwards(@config.base_dir_for_path_parameters) do |dir|
             # NOTE: Can't use `dir.glob` because of JRuby 9.4.8.0 incompatibility:
             # https://github.com/jruby/jruby/issues/8358
-            candidates = Pathname.glob("#{dir}/*.gemspec")
+            candidates = begin
+              Pathname.glob("#{dir}/*.gemspec")
+            rescue Errno::EACCES, Errno::EPERM
```

Comment:
Might be a good idea to add a comment mentioning we're ignoring the permission errors for scenarios like the one you've described in the ticket.